### ISSUE TITLE
Not sure how to force widescreen

### ISSUE THREAD

**Issue by HenWilOliPin** [m64p/mupen64plus-GLideN64#113]
Just started using m64p, and I can't figure out how to use widescreen. These 2 threads:
https://www.reddit.com/r/emulation/comments/3ezs6l/what_are_my_options_for_playing_n64_games_in/
https://www.reddit.com/r/EmulationOnAndroid/comments/4p7rb4/how_can_i_play_n64_games_in_widescreen/
say there are GlideN64 options I can use but I can't find them in the settings, and since the settings seem to handle a little differently than the other emulators I figured I'd try here. I'm playing DK64, if that's relevant.

**Comment by Wyatt100** [comment]
Go to Settings < Core and Plugin Settings < Video Plugin, and change AspectRatio to 0

**Comment by HenWilOliPin** [comment]
Yeah I figured it out, but I should clarify that I tried that before posting, and while it is technically "widescreen" it stretches out too much. Didn't notice there are tooltips at first, used them to learn that 2 forces 16:9 and 3 sets it to "adjust", which activated the widescreen hack in SM64 but not in DK64, so I'm assuming there is no hack for DK64 and I'll have to settle for 16:9
